code/__DEFINES/preferences.dm

```diff
@@ -36,7 +36,7 @@
 #define BE_ALIEN		64
 #define BE_PAI			128
 #define BE_CULTIST		256
-#define BE_NINJA		512
```

Comment:
We need to redo antag flags before we add any more, because we've reached the limit.